lib/LanguageServerCodeTransform/LspCommand/GenerateMethodCommand.php

```diff
@@ -61,6 +62,9 @@ public function __invoke(string $uri, int $offset): Promise
         } catch (MethodCallNotFound $error) {
             $this->clientApi->window()->showMessage()->warning($error->getMessage());
             return new Success(null);
+        } catch (CouldNotResolveNode $error) {
```

Comment:
i think both of these extend `NotFound` ?

Reply:
Indeed you are correct.